### DISCUSSION TITLE
Stable URL for latest release?

### DISCUSSION THREAD

**Discussion by sharpninja** [microsoft/winget-cli#606] [General]
Is there a stable URL I can point at with `Add-AppxPackage` to always get the latest version?

**Comment by DonEstefan** [comment]
Try https://github.com/microsoft/winget-cli/releases/latest/download/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.appxbundle
It worked for the last releases. This is unofficial and might stop working if devs decide to change their release process/names in the future (it is still beta after all).

**Reply by sharpninja** [reply]
That doesn't have the latest release.  I get a warning that I already have a newer version when installing from that link.

**Reply by DonEstefan** [reply]
v0.2.2941 is the latest "release" (which is what you asked for)
v0.2.3162 is available but labelled "pre-release" and not "release".
Maybe you already installed the pre-release?
Or maybe the message is just misleading and meant to say you have the "same or newer" version installed..?
